### ISSUE TITLE
My toilet is bricked 

### ISSUE THREAD

**Issue by xxmcvapourxx** [Karf5/Playstation-4-Special-Homebrew-Enabler#2]
I used your copy and paste job and used it on my toilet. You bricked it.... its all clogged up...fix it please.  Or I'll send an invoice to you for proper Devs to fix it.

**Comment by Prefixmodz** [comment]
Platinum Hen vibes

**Comment by DrYenyen** [comment]
Where have I went again I was looking for goldhen

![fef9e54da92e0d8c964ab76307d58ef6](https://github.com/user-attachments/assets/575f032b-4cf7-4ac0-b6c4-b7674fe0ac8a)

**Comment by Karf5** [comment]
If your toilet is broken, you should eat what is in it

**Comment by xxmcvapourxx** [comment]
> If your toilet is broken, you should eat what is in it

but I used your copy of hen whats different from  original the proper real dev. Your bricked my toilet

**Comment by Karf5** [comment]
My version of HEN provides the feature of blocking system updates, and I will add many more features in the future. But I don't understand what problem you are facing. If you really want to solve the problem you have, you should explain more

**Comment by HoppersPS4** [comment]
> My version of HEN provides the feature of blocking system updates, and I will add many more features in the future. But I don't understand what problem you are facing. If you really want to solve the problem you have, you should explain more

This is nothing new. GoldHEN has been doing this for ages. Nothing Special.

**Comment by Karf5** [comment]
My HEN is just a beta version now, and if you're paying attention, the HEN-VTX released by Echo Stretch does not have the feature of blocking updates. It is true that my HEN lacks many things, but I am working on improving it and adding more features to it

**Comment by xxmcvapourxx** [comment]
> My HEN is just a beta version now, and if you're paying attention, the HEN-VTX released by Echo Stretch does not have the feature of blocking updates. It is true that my HEN lacks many things, but I am working on improving it and adding more features to it

But it's not your hen you didn't write it from scratch all you did is download the hen uploaded it to github and added your shitty name to it 😉 we know your game.... you just steal people work hahaha 😆 😂

**Comment by Karf5** [comment]
Unfortunately, it seems that your sick and stupid minds are in a completely miserable state. You are a group of fools trying to frustrate people because you are failures. I was expecting better responses than that, but it seems that fools only write nonsense. What do you mean by your game? Don't you mean your game? You're trying to make me look like a thief, and You are trying to appear as someone who has value or knowledge of what you are saying, but like I said, fools only write nonsense and stupid things 😂😂😂

**Comment by DrYenyen** [comment]
> Unfortunately, it seems that your sick and stupid minds are in a completely miserable state. You are a group of fools trying to frustrate people because you are failures. I was expecting better responses than that, but it seems that fools only write nonsense. What do you mean by your game? Don't you mean your game? You're trying to make me look like a thief, and You are trying to appear as someone who has value or knowledge of what you are saying, but like I said, fools only write nonsense and stupid things 😂😂😂

If thief comes to mind for what you're doing then maybe do better and don't just copy paste things from people and then boast about it. When you add something of meaning then you can at least stand your ground.

**Comment by xxmcvapourxx** [comment]
> Unfortunately, it seems that your sick and stupid minds are in a completely miserable state. You are a group of fools trying to frustrate people because you are failures. I was expecting better responses than that, but it seems that fools only write nonsense. What do you mean by your game? Don't you mean your game? You're trying to make me look like a thief, and You are trying to appear as someone who has value or knowledge of what you are saying, but like I said, fools only write nonsense and stupid things 😂😂😂

Well at least you acknowledge that your a thief hahaa which is true. You stole hen renamed it and classed it as your own.. you're the only fool here numb nuts.... 🤣🤣🤣🤣 " look ma I'm a dev I've just downloaded hen and uploaded it to github and renamed it... by adding in my shitty name" 🙃

**Comment by DrYenyen** [comment]
> > Unfortunately, it seems that your sick and stupid minds are in a completely miserable state. You are a group of fools trying to frustrate people because you are failures. I was expecting better responses than that, but it seems that fools only write nonsense. What do you mean by your game? Don't you mean your game? You're trying to make me look like a thief, and You are trying to appear as someone who has value or knowledge of what you are saying, but like I said, fools only write nonsense and stupid things 😂😂😂
> 
> Well at least you acknowledge that your a thief hahaa which is true. You stole hen renamed it and classed it as your own.. you're the only fool here numb nuts.... 🤣🤣🤣🤣 " look ma I'm a dev I've just downloaded hen and uploaded it to github and renamed it... by adding in my shitty name" 🙃

Frrrr he forked it and acted like he did all the work behind it💀. Self awareness needs to be taught.

**Comment by Karf5** [comment]
As I said, a group of sick and stupid minds

**Comment by Prefixmodz** [comment]
smoll brain doesnt know what a pull request is

**Comment by xxmcvapourxx** [comment]
> As I said, a group of sick and stupid minds

no no we not a group... we a group that protects devs hard work. Where sick people like you who needs to look in the mirror and say to yourself. "Yeah they are right..I did steal the devs work. I should add credits and remove my name cause I've been caught out" that's the real truth.... YOU Lose mf..hahaha you didn't write the hen. You didn't do anything.. just add a little line that's it.

**Comment by Karf5** [comment]
As I said, a group of sick and stupid minds 🤣🤣

**Comment by Prefixmodz** [comment]
> As I said, a group of sick and stupid minds 🤣🤣

you could of just contributed to the original repo lmao

**Comment by xxmcvapourxx** [comment]
> As I said, a group of sick and stupid minds 🤣🤣

Yeah your the stupid mind 🙄 dude just get rid of it and start again... you will lose this battle...
